css/build.css

```diff
@@ -115,4 +115,72 @@
 
 .close-icon {
   width: 40%;
+}
+
+.fa-times-thin {
```

Comment:
Add a comment above this section to explain why this was necessary. Otherwise, future developers may find this code unnecessary and delete them.

Reply:
@tonytlwu Added a comment above `.fa-times-thin` section start.